### ISSUE TITLE
Colours Subtle, TextSpecial1, TextSpecial2 do not render with red channel < 128

### ISSUE THREAD

**Issue by d5xtgr** [RasmusLindroth/tut#6]
Colours are not correctly shown if the #RRGGBB representation has leading zeros (i.e. the red channel is less than 128).  This affects colours given as hex ("#008000") as well as by name ("green").

The following change fixes this behaviour.

```
--- util.go	2020-04-11 05:06:13.000000000 -0400
+++ util.go	2020-05-05 16:10:02.515224268 -0400
@@ -251,7 +251,7 @@
 }
 
 func ColorMark(color tcell.Color) string {
-	return fmt.Sprintf("[#%x]", color.Hex())
+	return fmt.Sprintf("[#%06x]", color.Hex())
 }
 
 func FormatUsername(a mastodon.Account) string {
```

**Comment by RasmusLindroth** [comment]
Thank you! Will include your fix when I get home from work :)

**Comment by RasmusLindroth** [comment]
Now it's merged in a99957207cec227991ee804fc395372d12e76fc8. Thanks again
